client/src/components/Input/Input.js

```diff
@@ -5,9 +5,10 @@ const Input = styled.input`
   border-radius: 50px;
   border: none;
   font-family: "Orbitron", Arial, sans-serif;
```

Comment:
You dont need it here, because you already declared it globally

```suggestion
```

Reply:
👍🏻 Thanks for review!